README.md

```diff
@@ -1,10 +1,10 @@
 OpenStreetMap Tile Attribution Issue Tracker
 ======================================
 
-This repository is used for reporting and tracking sites which are using tile.openstreetmap.org tiles but without attributing OpenStreetMap. The sites are tracked in the [issue tracker](https://github.com/openstreetmap/tile-attribution/issues).
```

Comment:
I think that introduction should be really clear that it is is about OSMF tile servers.

The OpenStreetMap standard layer is widely deployed on other servers, some of them used by entities not attributing OSM at all.